nautobot/dcim/models/devices.py

```diff
@@ -544,6 +545,20 @@ class Device(PrimaryModel, ConfigContextModel, StatusModel):
         blank=True,
         null=True,
     )
+    redundancy_group = models.ForeignKey(
+        to="RedundancyGroup",
+        on_delete=models.SET_NULL,
+        related_name="members",
+        blank=True,
+        null=True,
+    )
+    redundancy_group_priority = models.PositiveSmallIntegerField(
+        blank=True,
+        null=True,
```

Comment:
Allow null if `redundancy_group` set? Data validation for identical priorities?

Reply:
Validate/clean method to throw error/unset `redundancy_group_priority` if `redundancy_group` isn't set?